src/lib/deploy.js

```diff
@@ -419,7 +419,7 @@ async function deploy({ alias, yes }) {
 
   if (!txn || txn?.kind === 'error') {
     // Note that the thrown error object is already console logged via step().
-    log(red(getErrorMessage(txn.message ?? [])));
+    log(red(getErrorMessage(txn?.message ?? [])));
```

Comment:
this is an improvement, but I wonder if there's additional benefit in also looking at the `statusCode` and `statusText` (see what sendGraphQL returns) for potential info if the `txn.message` is not an array of errors. Also, I wonder if there could be more info in the `responseJson`

Reply:
That's a good point! Should we just dump the returned json if they are no array of errors to report on? On one hand, giving an `Unknown error` message is better than a runtime error but I would like to have more traceability in errors when they do occur.